CRM/MembershipExtras/Page/PaymentScheme.php

```diff
@@ -8,6 +8,13 @@ class CRM_MembershipExtras_Page_PaymentScheme extends CRM_Core_Page {
   public function run() {
     CRM_Utils_System::setTitle(E::ts('Payment Schemes'));
     $schemes = CRM_MembershipExtras_BAO_PaymentScheme::getAll();
+    foreach ($schemes as $key => $value) {
```

Comment:
@asem-compuco   Can you only load the list of payment processors one time? It is not necessary to call API every role to get payment processor name.